soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.series

```diff
@@ -10,4 +10,7 @@ source "soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.nrf52*"
 config SOC_SERIES
 	default "nrf52"
 
+config UART_USE_RUNTIME_CONFIGURE
+	default n
```

Comment:
Those are changes introduced in https://github.com/zephyrproject-rtos/zephyr/pull/65155 which this pull request is based on. Please comment in the original PR.

Reply:
nrf51 does not have uarte driver. Change is introduced because with nrfx uarte driver there is a significant flash size increase when runtime configuration is used (and it is almost never used). I added #65113 which enables this option whenever sample/module is using it.

Reply:
Unanswered

Reply:
Diff dropped from this PR.